### PR TITLE
About page

### PR DESCRIPTION
**Pull request by rstfari**
Basic template. No content so far.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+					Outer Ring Road, Devarabeesanahalli Village,
+
+					Bengaluru, 560103,
+
+					Karnataka, India</p>
+			</div>
+		</div>
+
+		<p id="footborder" style="max-width: 1000px;"></p>
```
Comment (GitWiz01) [inline about.html:224]: Align this footborder to center

**Comment by GitWiz01** [comment]
Please Add responsiveness
Screenshot 2023-12-25 220433

File: about.html (hunk 1 above)
Comment (rstfari) [reply about.html:224]: I could not for the life of me figure out how to align that footer. But I'll add responsiveness and comments I forgot

Comment (GitWiz01) [reply about.html:224]: Great go ahead!

**Comment by rstfari** [comment]
responsiveness should be fixed. Let me know if you need me to separate the css into its own file.

**Comment by GitWiz01** [comment]
> responsiveness should be fixed. Let me know if you need me to separate the css into its own file.

It will be great if u can do it

**Comment by rstfari** [comment]
separating the styles into its own file for some reason breaks the body.

**Comment by GitWiz01** [comment]
Thank you for the contribution

**Pull request merged**